x/supplier/types/message_stake_supplier_test.go

```diff
@@ -469,6 +507,34 @@ func TestMsgStakeSupplier_ValidateBasic(t *testing.T) {
 			},
 			expectedErr: ErrSupplierInvalidServiceConfig,
 		},
+		{
+			desc: "invalid service configs - owner cannot update",
```

Comment:
Is there a test where the service configs are empty AND signer by the owner?

Reply:
Yessir!

![image](https://github.com/user-attachments/assets/791bde1c-6e01-4dfa-9d8b-4a634f1dc61a)